integration-cli/docker_cli_cp_test.go

```diff
@@ -374,6 +374,8 @@ func TestCpUnprivilegedUser(t *testing.T) {
 }
 
 func TestCpVolumePath(t *testing.T) {
```

Comment:
It seems like only the bind-mounting parts of this are really "same host assuming", but that the rest of the "testing `docker cp`" logic is not only agnostic of that but would be useful client testing too -- do you think there'd be value in splitting this test instead?